src/nodes.rs

```diff
@@ -464,6 +480,7 @@ impl NodeValue {
             NodeValue::FootnoteReference(..) => "footnote_reference",
             #[cfg(feature = "shortcodes")]
             NodeValue::ShortCode(_) => "shortcode",
+            NodeValue::MultilineBlockQuote(_) => "block_quote",
```

Comment:
Happy to leave this as-is, but if you want to be able to roundtrip a blockquote's multilineness to XML[^xml], it'll need some distinction here.

[^xml]: which, let's face it, you probably don't rely on!

Reply:
I originally had it as `multiline_block_quote`, and then second guessed myself.  Thanks, I'll change it back.